layout/app/pulse/layer-pill/actions.js

```diff
@@ -24,8 +24,7 @@ export const toggleContextualLayer = createThunkAction('layer-pill/toggleContext
 
     if (!layerFound) {
       dispatch(setContextLayersLoading(true));
-      const layersService = new LayersService();
-      await layersService.fetchData({ id })
+      await dispatch(fetchLayer({ id }))
```

Comment:
Isn't the function paramater supposed to be `(id, params)` ? Being `id` an string, why do you send an object with a key `id` ?